### PR TITLE
added floor and ceil rounding with tests (#692)

### PR DESCRIPTION
**Pull request by simone-gasparini**
## Description
Added floor and ceil rounding with tests

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I have executed `bin/console kimai:phpcs --fix` to make sure my changes adopt the correct code style
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

File: tests/Timesheet/Rounding/CeilRoundingTest.php [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        $record->setDuration($record->getEnd()->getTimestamp() - $record->getBegin()->getTimestamp());
+
+        $sut = new CeilRounding();
+        $sut->roundBegin($record, $roundBegin);
+        $sut->roundEnd($record, $roundEnd);
+        $record->setDuration($record->getEnd()->getTimestamp() - $record->getBegin()->getTimestamp());
+        $sut->roundDuration($record, $roundDuration);
+
+        $this->assertEquals($expectedStart->getTimestamp(), $record->getBegin()->getTimestamp());
+//        $this->assertEquals($expectedEnd->getTimestamp(), $record->getEnd()->getTimestamp());
```
Comment (kevinpapst) [inline tests/Timesheet/Rounding/CeilRoundingTest.php]: why are these two lines commented?

Comment (simone-gasparini) [reply tests/Timesheet/Rounding/CeilRoundingTest.php]: My bad.. I fix it

**Comment by simone-gasparini** [comment]
Added documentation PR
https://github.com/kimai/www.kimai.org/pull/24

**Comment by simone-gasparini** [comment]
> Thanks for your PR! 👍
> 
> The new modes should be added that to the documentation:
> https://github.com/kimai/www.kimai.org/blob/master/_documentation/timesheet.md#rounding-of-begin-end-and-duration
> 
> I wonder why the tests don't have different numbers. There is something wrong, either the tests (what I guess) or the rounding mode.

The tests are based on the closest one, but with the correct results for the new implemented rounding

**Pull request merged**